index.html

```diff
@@ -72,9 +70,209 @@ <h3 class="discription-header">I'm a Software Developer</h3>
                 </li>
             </ul>
             <!--illustrations section-->
-            <div class="illustration-header" aria-hidden="true"></div>
-        </section>
-    </main>
-    <!-- main ends here -->
+            <div class="illustration-header" aria-hidden="true">   </div>
+    </section>
+        <!--work section starts here-->
+    <section class="work-section-container">
+        <div class="work-title-card">
+            <h2 class="work-title-header">My Recent Works</h2>
+
+            <svg alt="horizental line" class="title-separator" width="795" height="2" viewBox="0 0 795 2" fill="none"
+            xmlns="http://www.w3.org/2000/svg">
+            <path d="M794.5 1.00007L0.5 1" stroke="#212121" />
+          </svg>
+        </div>
+        <div class="work-card-section">
+            <img class="gym-fit-img" src="images/gym-fit-image.png" alt="gym-fit">
```

Comment:
- [ ] Great work including your gym post image here 💯. However, please kindly ensure you include the file `gym-fit-image.png` inside your `images` folder 🤓.
    >*Hint : I think I saw the missing image inside another branch called kiko. 🔧

- [ ] Also, please kindly delete the extra background image included in this section because it is not part of the Figma design. 🤓🔧

This is how your design looks 👇🏽

![design](https://user-images.githubusercontent.com/116701485/235292262-4c46d2d2-5147-41be-adc6-186fe06c1bdf.png)


This is the Figma design 👇🏽

![figma](https://user-images.githubusercontent.com/116701485/235292361-32c79ba8-1988-4f92-a392-d88c6e3484ce.png)